shopinvader_locomotive/data/ir_cron.xml

```diff
@@ -9,9 +9,8 @@
         <field name="interval_type">days</field>
         <field name="numbercall">-1</field>
         <field eval="False" name="doall"/>
-        <field eval="'shopinvader.backend'" name="model"/>
-        <field eval="'_scheduler_synchronize_currency'" name="function"/>
-        <field eval="'()'" name="args"/>
+        <field name="model_id" ref="model_shopinvader_backend"/>
+        <field name="code">model._scheduler_synchronize_currency()</field>
     </record>
 
     <record forcecreate="True" id="ir_cron_export_binding" model="ir.cron">
```

Comment:
I don't get why this is needed. Even if needed, shouldn't be placed into shopinvader main module?

Reply:
@simahawk, you're right. This one should go into shopinvader_search_engine.